src/commands/analyze/command.ts

```diff
@@ -486,6 +506,7 @@ export async function analyzeCommand(options: GenezioAnalyzeOptions) {
     // Inject Backend API URLs into the frontend component
     // This is done after all the components have been detected
     if (
+        !frameworksDetected.backend?.includes("genezio-typesafe") &&
```

Comment:
I think if you reverse these conditions you won't need `?` anymore. Also, it would be more natural to read.

```
 if (
        frameworksDetected.backend &&
        frameworksDetected.backend.length > 0 &&
        !frameworksDetected.backend.includes("genezio-typesafe") &&
        frameworksDetected.frontend &&
```